subsys/bluetooth/common/Kconfig

```diff
@@ -183,12 +189,44 @@ config BT_BUF_CMD_TX_SIZE
 	  to the maximum of 255.
 
 config BT_BUF_CMD_TX_COUNT
-	int "Number of HCI command buffers"
-	default 2
-	range 2 64
+	int "Number of HCI command buffers" if !BT_HCI_RAW || !HAS_BT_CTLR
+	# !BT_HCI_ACL_FLOW_CONTROL ==> Permit configurable Num_HCI_Command_Packets for
+	# Host-only build but hidden when HAS_BT_CTLR=y (default 1 for Controller-only build).
+	# !(BT_HCI_RAW && HAS_BT_CTLR) ==> Host-only build.
+	depends on !BT_HCI_RAW || !HAS_BT_CTLR || !BT_HCI_ACL_FLOW_CONTROL
+	# BT_MESH is starting Extended Advertising in the callback run in the system work queue and
+	# as the Host also uses the system work queue for Tx towards the Controller, one additional
+	# HCI command buffer is needed.
+	range 2 64 if BT_MESH
```

Comment:
Can you elaborate on this? 

`system work queue for Tx towards the Controller` - TX of what? 
Aren't the HCI commands sent synchronously?